pkg/common/catalog/interfaces/context.go

```diff
@@ -0,0 +1,14 @@
+package interfaces
+
+import "context"
+
+type pluginNameKey struct{}
+
+func PluginNameFromHostServiceContext(ctx context.Context) (string, bool) {
```

Comment:
it is not an interface, maybe we can move to another package?

Reply:
I'd rather rename the package than spread this stuff out. I'll think of a better name.

Reply:
Renamed to internal.